runner/runners/invoke.go

```diff
@@ -131,17 +134,16 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 
 	switch st.State {
 	case execer.COMPLETE:
-		srcToDest := map[string]string{
-			stdout.AsFile(): "STDOUT",
-			stderr.AsFile(): "STDERR",
+		tmp, err := temp.NewTempDir("", "invoke")
```

Comment:
If this is running on a worker that runs on Mesos and I think it is, we want to make sure we _never_ use the system temp dir, and be sure that everything we create is in our task sandbox ("./" more or less). By passing "" we are using os.TempDir, which is bad and we should scrub these from our repo.